app/frontend/src/features/auth/AuthPage.tsx

```diff
@@ -47,61 +62,145 @@ const useStyles = makeStyles((theme) => ({
     justifyContent: "space-around",
-    marginTop: theme.spacing(3),
+    marginTop: theme.spacing(3, 0),
+  },
+  signupLink: {
+    backgroundColor: theme.palette.primary.main,
+    borderRadius: "4px",
+    color: "#ffffff",
+    fontWeight: 500,
+    fontSize: "20px",
+    padding: theme.spacing(1),
+    textAlign: "center",
+    textDecoration: "none",
+    width: "88px",
   },
   subtitle: {
     marginTop: theme.spacing(1),
+    [theme.breakpoints.up("md")]: {
+      display: "inline-block",
+      marginTop: theme.spacing(5),
+      position: "relative",
+    },
+  },
+  title: {
+    ...theme.typography.h1,
+    [theme.breakpoints.up("md")]: {
+      fontSize: "64px",
+      lineHeight: "74px",
+      textAlign: "left",
+    },
+  },
+  underline: {
+    borderTop: `5px solid ${theme.palette.primary.main}`,
+    boxShadow: "0px 4px 4px rgba(0, 0, 0, 0.25)",
+    left: theme.spacing(1),
+    position: "absolute",
+    width: "100%",
   },
-  title: theme.typography.h1,
 }));
 
 export default function AuthPage() {
   const classes = useStyles();
   const authClasses = useAuthStyles();
 
   return (
-    <Box className={classes.authPage}>
-      <Box className={classes.content}>
-        <Typography variant="h1">{INTRODUCTION}</Typography>
-        <Typography classes={{ root: classes.subtitle }}>
-          {COLLABORATIVE}
-        </Typography>
-        <Divider classes={{ root: authClasses.divider }} flexItem />
-        <Box className={classes.navigation}>
-          <Button
-            component={Link}
-            to={loginRoute}
-            classes={{
-              label: authClasses.buttonText,
-              root: `${authClasses.button} ${classes.button}`,
-            }}
-            color="secondary"
-          >
-            {LOGIN}
-          </Button>
-          <Button
-            component={Link}
-            to={signupRoute}
-            classes={{
-              label: authClasses.buttonText,
-              root: `${authClasses.button} ${classes.button}`,
-            }}
-            color="secondary"
-          >
-            {SIGN_UP}
-          </Button>
-        </Box>
-        <Box className={classes.aboutUs}>
-          <Typography className={classes.aboutUsLink} component={Link} to="#">
-            {ABOUT_US}
+    <div className={classes.authPage}>
+      <Hidden mdUp>
+        <div className={classes.content}>
+          <Typography variant="h1">{INTRODUCTION}</Typography>
+          <Typography classes={{ root: classes.subtitle }}>
+            {COLLABORATIVE}
           </Typography>
-          <ExpandMoreIcon />
-        </Box>
-      </Box>
-    </Box>
+          <Divider classes={{ root: authClasses.divider }} flexItem />
+          <div className={classes.mobileNavigation}>
+            <Button
+              component={Link}
+              to={loginRoute}
+              classes={{
+                label: authClasses.buttonText,
+                root: `${authClasses.button} ${classes.button}`,
```

Comment:
I will accept that `label: ` is better than using `className` with a fancy selector like `"& .MuiButtonLabel"` or something. But for `root`, maybe use `className`. I'll ask on slack.

Reply:
Agreed!